nlp-model.py

```diff
@@ -0,0 +1,52 @@
+import re
+from nltk.stem import WordNetLemmatizer
+nltk.download('wordnet')
+lemmatizer = WordNetLemmatizer()
+
+
+class PreprocessText(object):
```

Comment:
When you're creating a class in Python, it inherits from the `object` base class by default. Such that writing `class PreprocessText(object):` is equivalent to `class PreprocessText:`. The trade-off being between explicitness and brevity, I think we could go either way with this, thoughts?

Reply:
```suggestion
class PreprocessText:
```